src/components/TokenSelector.jsx

```diff
@@ -6,27 +6,35 @@ import Checkbox from 'material-ui/Checkbox';
 class TokenSelector extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {
```

Comment:
You should be using `this.state` and `this.setState` to manage the state of the component